### PR TITLE
Added New Vulnerability Search Website

### PR DESCRIPTION
**Pull request by RahulPratapSingh**
Added New Vulnerability Search Website in default.cfg and default_backtrack.cfg

### CODE REVIEW

File: profiles/resources/default.cfg [hunk 1]
```diff
@@ -6,6 +6,7 @@ VulnSearch_____ExploitSearch (Exploits Only)_____http://www.exploitsearch.net/in
 VulnSearch_____ExploitSearch (All)_____http://www.exploitsearch.net/index.php?q=@@@PLACE_HOLDER@@@
 VulnSearch_____NVD (All)_____http://web.nvd.nist.gov/view/vuln/search-results?query=@@@PLACE_HOLDER@@@&search_type=all&cves=on
 VulnSearch_____OSVDB (All)_____http://osvdb.org/search/search?search%5Bvuln_title%5D=@@@PLACE_HOLDER@@@&search%5Btext_type%5D=alltext&search%5Bs_date%5D=&search%5Be_date%5D=&search%5Brefid%5D=&search%5Breferencetypes%5D=&search%5Bvendors%5D=&search%5Bcvss_score_from%5D=7&search%5Bcvss_score_to%5D=10&search%5Bcvss_av%5D=*&search%5Bcvss_ac%5D=*&search%5Bcvss_a%5D=*&search%5Bcvss_ci%5D=*&search%5Bcvss_ii%5D=*&search%5Bcvss_ai%5D=*&kthx=search
+VulnSearch_____ExploitSearch_____https://packetstormsecurity.com/search/?q=@@@PLACE_HOLDER@@@
```
Comment (tunnelshade) [inline profiles/resources/default.cfg]: Name it `PacketStorm Exploit search`. See other search website names, they actually include the service name in it

**Comment by RahulPratapSingh** [comment]
@tunnelshade Renamed it :)

**Comment by delta24** [comment]
@tunnelshade looks good to merge?

**Comment by tunnelshade** [comment]
@delta24 Not yet!!! Need some more info on what all filters packetstorm provides!

**Comment by delta24** [comment]
@RahulPratapSingh can you check if there are any more filter parameters that can be added?

**Comment by RahulPratapSingh** [comment]
@delta24 I checked on that site, search for cve-id can be done.
https://packetstormsecurity.com/files/cve/CVE-ID

**Comment by delta24** [comment]
@RahulPratapSingh are you still working on this?

**Comment by RahulPratapSingh** [comment]
@delta24 I have checked the filters. I just found two filters on packetstorm.
1) General Search
2) Search for CVE-ID

**Comment by delta24** [comment]
@RahulPratapSingh can you add those filters like others in the resources.cfg?

**Comment by delta24** [comment]
@RahulPratapSingh  can you resend the PR? I have removed the Backtrack files from version control.

**Pull request closed without merging**